src/grelu/interpret/motifs.py

```diff
@@ -246,7 +247,7 @@ def score_sites(
 
     # Score sites for each sequence
     df["site_attr_score"] = df.apply(
-        lambda row: attrs[row.seq_idx, :, row.start : row.end].mean(), axis=1
+        lambda row: attrs[row.seq_idx, :, row.start : row.end].sum(0).mean(), axis=1
```

Comment:
@ketringjoni I don't have an issue with the `.sum(0)` change but it is breaking one of the tests (https://github.com/Genentech/gReLU/blob/main/tests/test_interpret.py#L246). Can we remove it from this PR and make the change in a separate PR where we also modify the tests?